.github/ISSUE_TEMPLATE/config.yml

```
@@ -4,7 +4,7 @@ contact_links:
     url: https://github.com/orgs/supabase/discussions/categories/feature-requests?discussions_q=is%3Aopen+category%3A%22Feature+Requests%22+sort%3Atop
     about: Request a new feature or example.
   - name: Ask a question
-    url: https://github.com/supabase/supabase/discussions/category_choices
+    url: https://github.com/orgs/supabase/discussions/categories/questions
```

@dshukertjr

> In that case, I think the link is taking the user to the intended destination. The user can get started with composing their questions right away from that page!

I see. Shall I change it this way to suppress that notice?

```suggestion
    url: https://github.com/orgs/supabase/discussions/new/choose
```

---

Oh yeah, that would be amazing!